File: Finanzuebersicht.Frontend.Web.Admin/src/app/model/accounting/accounting-entries/dtos/i-accounting-entry-create.ts
export interface IAccountingEntryCreate {
    categoryId: string;
    auftragskonto: string;
    buchungsdatum: Date;
    valutaDatum: Date;
    buchungstext: string;
    verwendungszweck: string;
    glaeubigerId: string;
    mandatsreferenz: string;
    sammlerreferenz: string;
    lastschriftUrsprungsbetrag: number;
    auslagenersatzRuecklastschrift: string;
    beguenstigter: string;
    iban: string;
    bic: string;
    betrag: number;
    waehrung: string;
    info: string;
}

File: Finanzuebersicht.CodeGeneration/2-Post-Contractor/Backend/Contract/Logic/Modules/Accounting/AccountingEntries/DTOs/IAccountingEntryCreate.cs
using System;

namespace Finanzuebersicht.Backend.Generated.Contract.Logic.Modules.Accounting.AccountingEntries
{
    public interface IAccountingEntryCreate
    {
        Guid CategoryId { get; set; }

        string Auftragskonto { get; set; }

        DateTime Buchungsdatum { get; set; }

        DateTime ValutaDatum { get; set; }

        string Buchungstext { get; set; }

        string Verwendungszweck { get; set; }

        string GlaeubigerId { get; set; }

        string Mandatsreferenz { get; set; }

        string Sammlerreferenz { get; set; }

        double LastschriftUrsprungsbetrag { get; set; }

        string AuslagenersatzRuecklastschrift { get; set; }

        string Beguenstigter { get; set; }

        string IBAN { get; set; }

        string BIC { get; set; }

        double Betrag { get; set; }

        string Waehrung { get; set; }

        string Info { get; set; }
    }
}